### PR TITLE
truffle/npm import support

### PR DESCRIPTION
**Pull request by 0v1se**
according to http://truffleframework.com/docs/getting_started/compile
truffle supports npm packages (see import "somepackage/SomeContract.sol")

this PR adds import file resolve for npm

### CODE REVIEW

File: src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt [hunk 1]
```diff
@@ -13,13 +14,27 @@ class SolImportPathReference(element: SolImportPathElement) : SolReferenceBase<S
       return null
     }
     val path = importText.substring(1, importText.length - 1)
-    val file = element.containingFile.virtualFile.findFileByRelativePath("../$path")
+    val virtualFile = element.containingFile.virtualFile
+    val file = if (path.startsWith(".")) {
```
Comment (SerCeMan) [inline src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt]: It seems like relative `path` doesn't always start with a `.`, e.g. I've seen code like
```
import "oraclize/usingOraclize.sol";

contract PriceTicker is usingOraclize, Owned {
```

But I think it's possible to use `node_modules` search as a fallback.

Comment (evgenynacu) [reply src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt]: Although, Truffle doesn't work when you use something like 'import "oraclize/usingOraclize.sol"'
But, users can use some other framework or compile directly.

I can close the PR and use my implementation as fallback.
Then will resubmit it

Comment (0v1se) [reply src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt]: Although, Truffle doesn't work when you use something like 'import "oraclize/usingOraclize.sol"'
But, users can use some other framework or compile directly.

I can close the PR and use my implementation as fallback.
Then will resubmit it. OK?

**Comment by SerCeMan** [comment]
👍  LGTM, I'll merge it once CI is happy, travis seems to be a bit sloppy today

**Pull request merged**